.mergify.yml

```diff
@@ -0,0 +1,146 @@
+# This file was automatically generated by sbt-typelevel-mergify using the
+# mergifyGenerate task. You should add and commit this file to
+# your git repository. It goes without saying that you shouldn't edit
+# this file by hand! Instead, if you wish to make changes, you should
+# change your sbt build configuration to revise the mergify configuration
+# to meet your needs, then regenerate this file.
+
+pull_request_rules:
+- name: merge scala-steward's PRs
+  conditions:
+  - author=scala-steward
+  - or:
+    - body~=labels:.*early-semver-patch
+    - body~=labels:.*early-semver-minor
+  - status-success=Build and Test (ubuntu-latest, 2.12.15, temurin@8, rootJVM)
+  - '#approved-reviews-by>=1'
+  actions:
+    merge: {}
+- name: Label ci PRs
+  conditions:
+  - files~=^ci
+  actions:
+    label:
+      add:
+      - ci
+      remove: []
+- name: Label ci-release PRs
+  conditions:
+  - files~=^ci-release
+  actions:
+    label:
+      add:
+      - ci-release
+      remove: []
+- name: Label ci-signing PRs
+  conditions:
+  - files~=^ci-signing
+  actions:
+    label:
+      add:
+      - ci-signing
+      remove: []
+- name: Label core PRs
+  conditions:
+  - files~=^core
+  actions:
+    label:
+      add:
+      - core
+      remove: []
+- name: Label github PRs
+  conditions:
+  - files~=^github
```

Comment:
Hmm, this should end in `/`, or else it will pick up changes in `github-actions` as well.